tests/unit/core/test_tracking.js

```diff
@@ -279,6 +291,81 @@ describe(__filename, () => {
         getAddonEventCategory(ADDON_TYPE_STATIC_THEME, INSTALL_ACTION),
       ).toEqual(INSTALL_THEME_CATEGORY);
     });
+
+    it('returns the expected category when type is extension and installAction is cancelled', () => {
+      expect(
+        getAddonEventCategory(ADDON_TYPE_EXTENSION, INSTALL_CANCELLED_ACTION),
+      ).toEqual(INSTALL_EXTENSION_CANCELLED_CATEGORY);
+    });
+
+    it('returns the expected category when type is lightweight theme and installAction is cancelled', () => {
+      expect(
+        getAddonEventCategory(ADDON_TYPE_THEME, INSTALL_CANCELLED_ACTION),
+      ).toEqual(INSTALL_THEME_CANCELLED_CATEGORY);
+    });
+
+    it('returns the expected category when type is static theme and installAction is cancelled', () => {
```

Comment:
I don't see this option to cancel with static theme, do you?

Reply:
sorry just to clarify I see the following for extension (Add AND Cancel options):

![Alt text](https://monosnap.com/image/K7mtdk1Xh27ZTDDRyyfue2Hf09Xya7.png)

but I wasn't see this for any of the themes. If you see these then never mind..

Reply:
ooh right!

Reply:
indeed, there is no button to cancel a theme installation (lightweight or static theme), but we have to cover this by test, no?

Reply:
yep, I see...that makes sense. thanks : )